client/app/main/main.html

```diff
@@ -26,38 +22,41 @@
 
 <div class='what-is'>
   <div class='container'>
-    <div class='col-md-6 icons'>
-      <span class='glyphicon glyphicon-phone'></span> +
-      <span class='glyphicon glyphicon-book'></span>
-    </div>
-    <div class='col-md-6'>
-      <h3>What is venue?</h3>
-      venue is a platform that allows instructors to give students credit for
-      attending events. An instructor signs on and creates courses and events,
-      students can then use the venue app to upload event submissions, which
-      can contain images or GPS data that verifies a student attended an event.
+    <div class='row'>
+      <div class='col-md-6 icons'>
+        <span class='glyphicon glyphicon-phone'></span><span class='plus-text'> + </span>
+        <span class='glyphicon glyphicon-book'></span>
+      </div>
+      <div class='col-md-6'>
+        <h3>What is venue?</h3>
```

Comment:
Josh recommended we stay consistent with a capital V, I think I agree (logo stays lowercase tho) :penguin: